docs/onboarding/08-logging.md

```diff
@@ -0,0 +1,35 @@
+# Chapter 8: Logging
```

Comment:
Should we name this chapter Monitoring instead of Logging.

Reply:
I would have loved to but I instead narrowed the scope to just include logging because monitoring includes both logging and alerting.

Reply:
I'll update the doc by calling this out.